kmq.js

```diff
@@ -78,6 +94,22 @@ const parseMessage = (message, botPrefix) => {
         console.error("No bot token set. Please update config.json!")
         process.exit(1);
     }
+    let guildPreferencesTableCreation = `CREATE TABLE IF NOT EXISTS guildPreferences(
```

Comment:
Convention is underscore-case instead of camelcase

Reply:
this is factual